Lib/fontmake/__main__.py

```diff
@@ -360,6 +360,15 @@ def main(args=None):
         action="store_false",
         help="Do not auto-generate a GDEF table, but keep an existing one intact.",
     )
+    outputGroup.add_argument(
+        "--save-ufo-as-zip",
+        dest="ufo_structure",
+        action="store_const",
+        const="zip",
+        default="package",
+        help="Save UFOs as .ufoz format. Only valid when generating UFO masters "
```

Comment:
actually one does not "generate UFO masters from designspace", they are the sources themselves. Maybe you meant "from .glyphs"

Reply:
i'll change the wording myself, thanks for the PR